### DISCUSSION TITLE
板绘难度高吗

### DISCUSSION THREAD

**Discussion by 202426710** [share121/inter-knot#3189] [其他]
打算学习板绘自产自销，有没有懂得大佬给点建议，本人零基础

**Comment by Nie7bai** [comment]
emmm……肿么说呢，有绘画基础的话，并且有那种带屏的绘板或者平板的话，一般学习一下如何使用绘画软件就行了
如果没有屏的的绘板，上手难度还是有点高的，要手眼同步，适应一下
没有绘画基础的话，板绘其实上手难度并不是很好，多练习就好
电脑的软件可以用PS（好像要花钱，我用的盗版的<img src="https://img-static.mihoyo.com/communityweb/upload/0a74f8be80231a76381738498edd52a6.png" alt="三月七_尴尬" width="75" >）、SAI（这个也挺好用的，好像免费？忘了）
平板或者手机的软件可以用画世界（还有，但我忘了）
反正加油就是了，慢慢画<img src="https://img-static.mihoyo.com/communityweb/upload/408018316100a8a8a846c68acedaa191.gif" alt="米游姬 卖萌" width="75" >

**Reply by 202426710** [reply]
谢谢(●'◡'●)
